File: 01977.py
from math import sqrt

M = int(input())
N = int(input())

sum = 0
for i in range(N, M-1, -1):
    if int(sqrt(i)) ** 2 == i:
        sum += i
        m = i

if sum == 0: print(-1)
else: print(sum, m, sep='\n')

File: 01977.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
	int M, N; cin >> M >> N;

	int sum = 0, min;
	for (int i = N; i >= M; i--) {
		if ((int)sqrt(i) * (int)sqrt(i) == i) {
			sum += i;
			min = i;
		}
	}

	if (!sum) cout << -1 << endl;
	else cout << sum << '\n' << min << endl;
}
